scripts/third_party_size_check.py

```diff
@@ -21,10 +21,21 @@
 
 import os
 import sys
+import yaml
```

Comment:
Hi @apb7 yes it is modules and subset of PyYAML, we have used it before also [here](https://github.com/oppia/oppia/blob/develop/utils.py) and we have installed it for Travis as well [here](https://github.com/oppia/oppia/blob/develop/.travis.yml) (Line: 97)

Reply:
Great, thanks for the links. Just wondering -- did this import order (and the PR in general) pass the pre-submit checks? I see that you have not ticked this checkbox above in the PR explanation.

Reply:
yes, the import order passed.
but `pre_commit_linter.py` failed because the PR is not within the coding guidelines and I was not sure with the implementation of the same.

Reply:
OK, please fix all such issues. Also, from the future, please adhere to the coding guidelines even if you're unsure about the implementation and ensure that pre-submit checks always pass before making a PR :)

Reply:
yes sure, I'll do Thanks!